### PR TITLE
Machine Learning task

### PR DESCRIPTION
**Pull request by soh456**

### CODE REVIEW

File: Machine Learning/Task Submission/Sohum.ipynb [hunk 1]
```diff
@@ -0,0 +1,572 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset = pd.read_csv(\"iris_flowers.csv\")"
+   ]
+  },
```
Comment (soh456) [inline Machine Learning/Task Submission/Sohum.ipynb:31]: reading the dataset

File: Machine Learning/Task Submission/Sohum.ipynb [hunk 2]
```diff
@@ -0,0 +1,572 @@
...
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dataset[\"class\"].value_counts()"
+   ]
+  },
```
Comment (soh456) [inline Machine Learning/Task Submission/Sohum.ipynb:360]: Previewing the data

File: Machine Learning/Task Submission/Sohum.ipynb [hunk 3]
```diff
@@ -0,0 +1,572 @@
...
+     "output_type": "stream",
+     "text": [
+      "(150,)\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(y.shape)"
+   ]
+  },
```
Comment (soh456) [inline Machine Learning/Task Submission/Sohum.ipynb:449]: Modeling with scikit-learn

File: Machine Learning/Task Submission/Sohum.ipynb [hunk 4]
```diff
@@ -0,0 +1,572 @@
...
+     "output_type": "stream",
+     "text": [
+      "0.9733333333333334\n"
+     ]
+    }
+   ],
+   "source": [
+    "#before spliting accuracy of prediction \n",
+    "logreg = LogisticRegression()\n",
+    "logreg.fit(X, y)\n",
```
Comment (Dev-Neon221) [inline Machine Learning/Task Submission/Sohum.ipynb:478]: You don't need to use the whole dataset unnecessarily to train the model. First split it to train and test data.
And only use train data to train the model.

**Pull request merged**